sidebar.json

```diff
@@ -459,6 +459,46 @@
                      }
                   ]
                },                              
+               {
+                  "title": "Blazor",
+                  "children": [
+                     {
+                        "title": "Creating your First Chart",
+                        "children": [
+                           {
+                              "title": "Your First Chart",
+                              "path": "/getting-started/blazor/your-first-chart-using-blazor"
+                           },
+                           {
+                              "title": "Flutter Features",
+                              "path": "/getting-started/blazor/blazor-features"
+                           },
+                           {
+                              "title": "Rendering Different Charts",
+                              "path": "/getting-started/blazor/rendering-different-charts-using-blazor"
+                           }
+                        ]
+                     },
+                     {
+                        "title": "Usage Guide",
+                        "children": [
+                           {
+                              "title": "Working with Events",
+                              "children": [
+                                 {
+                                    "title": "Change Chart Type",
+                                    "path": "/getting-started/blazor/change-chart-type-using-blazor"
+                                 },
+                                 {
+                                    "title": "Apply Different Themes",
+                                    "path": "/getting-started/blazor/apply-different-themes-using-blazor"
```

Comment:
I do not think, we have a working example in the repository as to how this can be done. Not that it cannot be done but considering we do not have a sample readily, we should focus on other parts of the blazor. let me know what do you think?

Reply:
Updated, page was removed.